.env.example

```diff
@@ -13,3 +14,4 @@ AZURE_SEARCH_SUMMARIES_INDEX_NAME=
 
 AZURE_OPENAI_API_INSTANCE_NAME=
 AZURE_OPENAI_API_KEY=
+AZURE_OPENAI_MODEL_NAME=
```

Comment:
Do we need to add this to snd3/4?

Reply:
Have now added model name to snd3/4 config